src/kernels.ts

```diff
@@ -1,17 +1,17 @@
 import { NotebookPanel } from '@jupyterlab/notebook';
 
 export const KERNEL_URL_TO_NAME: Record<string, string> = {
-  python: 'xpython',
+  python: 'python',
   r: 'xr'
 };
 
 export const KERNEL_NAME_TO_URL: Record<string, string> = {
-  xpython: 'python',
```

Comment:
Arguably we can leave an `xpython` entry here, so that in the future it is compatible with either the xeus-python or pyodide kernels.